src/test/resources/caseDetailsScotTest1.json

```diff
@@ -195,6 +195,7 @@
     "responseReceived": "none",
     "userLocation": "Liverpool",
     "hearingNumbers" : "2",
+    "managingOffice" : "Manchester",
```

Comment:
Does it make sense to set the managingOffice for a Scotland caseDetails to value that doesn't exist for Scotland?